Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -283,11 +387,16 @@ public Task<TResult> InvokeWithParameterObjectAsync<TResult>(string targetName,
             await InvokeDidChangeConfigurationAsync(config);
```

Comment:
minor. just noticed this `await ` isn't needed. could just return `InvokeDidChangeConfigurationAsync`

Reply:
Actually it is. Well unless I'm misunderstanding how C# tasks work. There's an await at the top of this function. AFAICT that means I have to await the task on exit too? At least when I change this to a return, it gives me a compiler error:

![image](https://user-images.githubusercontent.com/19672699/113603072-5119b780-95f8-11eb-9e33-5f2d648b2b00.png)

Reply:
i guess because its not an async Task then?

Reply:
No I don't think it's checking that. 

It's if you have an async applied to a function, you cannot return a task. You have to return the type of the task and in this case the Task is of type void, so you cannot return anything.